### PR TITLE
Return the script 'make-coverage.py' to sandbox 

### PR DESCRIPTION
**Pull request by SherineAwad**

### CODE REVIEW

**Comment by mr-c** [comment]
@SherineAwad & @ctb given the recent email on this topic it would be nice to re-include this script for the next release

**Comment by SherineAwad** [comment]
@mr-c  @ctb Ready for review: please take a look.

File: sandbox/README.rst [hunk 1]
```diff
@@ -25,6 +25,7 @@ Scripts with recipes:
 
 To keep, document, and build recipes for:
 
+* `make-coverage.py <https://github.com/ged-lab/khmer/tree/691b0b3aefe83e9e8f5f2b80f5f9516664a4654a/sandbox/make-coverage.py>`__ - RPKM calculation script
```
Comment (mr-c) [inline sandbox/README.rst]: No need for the link here.

Comment (SherineAwad) [inline sandbox/README.rst]: @mr-c I removed the link. Ready for review.

**Comment by mr-c** [comment]
@SherineAwad I had a hard time following all the commits. I've squashed this down to one commit for review purposes.

**Comment by mr-c** [comment]
Jenkins, test this please

**Comment by mr-c** [comment]
Jenkins, test this please

**Comment by mr-c** [comment]
@SherineAwad I am trying to merge your pull request; please do not push.

**Comment by mr-c** [comment]
Thank you @SherineAwad

**Pull request merged**